### PR TITLE
[AN-4513] [AN-4425] Forbid the user to set an empty user name or input line breaks

### PR DESCRIPTION
**Pull request by JPonte**
Tickets:
https://wearezeta.atlassian.net/browse/AN-4513
https://wearezeta.atlassian.net/browse/AN-4425

#### APK
[Download build #7790](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7790/artifact/build/artifact/wire-dev-PR258-7790.apk)
[Download build #7792](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7792/artifact/build/artifact/wire-dev-PR258-7792.apk)
[Download build #7793](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7793/artifact/build/artifact/wire-dev-PR258-7793.apk)

### CODE REVIEW

**Comment by mykola-mokhnach** [comment]
There was a problem, that it is possible to insert line breaks into user name. This should not be allowed.

**Comment by mykola-mokhnach** [comment]
- Minimum name length should be two characters. Everything, which is less than that, is incorrect.
- The input cursor should be automatically moved to the end of name text if I click "Name" item in Options. Now the cursor is placed at the start of the input field, so I have to manually tap the field to be able to delete characters with backspace button:
cursor

**Comment by mykola-mokhnach** [comment]
Cannot get the build #7769  from Jenkins

**Comment by xuanzhaopeng** [comment]
@JPonte  Cannot get 7778 build

**Comment by hakonbo** [comment]
build again

**Comment by hakonbo** [comment]
build again

File: wire-core/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -486,7 +488,7 @@
     <string name="pref_options_spotify_title">Connect with Spotify</string>
     <string name="pref_options_spotify_logout_title">Disconnect with Spotify</string>
     <string name="pref_options_spotify_summary">You need a Spotify Premium account.</string>
-    <string name="pref_options_spotify_login_failed">Login failed. Please try again!</string>
+    <string name="pref_options_spotify_login_failed">Login failed. Please try again!</string>"
```
Comment (hakonbo) [inline wire-core/src/main/res/values/strings.xml]: Seems like an extra ```"``` was added here...

**Pull request merged**